src/modes/exportdata/ExportData.svelte

```diff
@@ -27,18 +27,37 @@
     }
   }
 
+  // Pre-fill the form from URL parameters.
+  // Supported parameters: data_source, signal, start_day, end_day, geo_type, geo_value, as_of
+  const urlParams = new URLSearchParams(window.location.search);
+
+  // Overwrite default source & sensor if these are present in the URL
+  if (urlParams.has('data_source')) {
+    sourceValue = urlParams.get('data_source');
+
+    if (urlParams.has('signal')) {
+      sensorValue = urlParams.get('signal');
+    }
+  }
+
   $: isWeekly = sensor ? sensor.isWeeklySignal : false;
   $: formatDate = isWeekly ? (d) => formatWeek(d).replace('W', '') : formatDateISO;
 
-  let geoType = 'county';
+  let geoType = urlParams.has('geo_type') ? urlParams.get('geo_type') : 'county';
 
   let startDate = new Date();
   let endDate = new Date();
 
   function initDate(date) {
     const param = new DateParam(date);
-    endDate = param.sparkLineTimeFrame.max;
-    startDate = param.sparkLineTimeFrame.min;
+
+    // Populate date based on URL params or, if absent, the current date
+    startDate = urlParams.has('start_day') ? new Date(urlParams.get('start_day')) : param.sparkLineTimeFrame.min;
+    endDate = urlParams.has('end_day') ? new Date(urlParams.get('end_day')) : param.sparkLineTimeFrame.max;
+
+    // Also normalize the dates to the current timezone
+    startDate = new Date(startDate.getTime() - startDate.getTimezoneOffset() * -60000);
+    endDate = new Date(endDate.getTime() - endDate.getTimezoneOffset() * -60000);
```

Comment:
Let's add a comment about the Javascript string parsing + negative timezone edge cases here. If this is a common fix, maybe we can include a link instead of writing the full explanation out.